core/src/main/java/org/opensearch/sql/data/model/ExprTimestampValue.java

```diff
@@ -30,7 +30,7 @@ public class ExprTimestampValue extends AbstractExprValue {
   /**
    * todo. only support UTC now.
    */
-  private static final ZoneId ZONE = ZoneId.of("UTC");
+  public static final ZoneId ZONE = ZoneId.of("UTC");
```

Comment:
just wondering should we get this info from `FunctionProperties` ?

Reply:
That means we have to deliver `FunctionProperties` everywhere `ExprTimestampValue` created or used. It is too big change to be in this PR.

Reply:
Sure, just curious about how to handle this in future. I recall there are info in SQL session such as current catalog/schema, user/role, time zone, language etc. Probably we need to think about how to do this with FunctionProperties or a new Session abstraction. Thanks!

Reply:
Definitely want to do this with FunctionProperties if we can.  If we end up sending tasks to multiple nodes with query details, we will want to include the execution start time (in FunctionProperties) as well as the zone id (again, in FunctionProperties).  The design of the FunctionProperties is that it is expandable to contain all the query execution details. 
cc: @MaxKsyunz